payara-micro/demystifying-microservices-example/CrudView/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fish.payara</groupId>
+    <artifactId>CrudView</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+    <name>CrudView</name>
+    <parent>
```

Comment:
Just for convention's sake would you mind putting the parent at the top?

Reply:
Checked a few other pom files in the repo, all of them have the <parent> tag under name, seems to be consistent with other pom files. Leaving as-is for now.

See:
microprofile/pom.xml
microprofile/rest-client/pom.xml
payara-micro/jpa-datasource-example/pom.xml